packages/apps/hr/src/screens/ExpenseListScreen.js

```diff
@@ -0,0 +1,226 @@
+    return Expense.getStatusList(Colors, I18n);
+  }, [Colors, I18n]);
+
+  const fetchExpenseAPI = useCallback(
+    (page = 0) => {
+      dispatch(searchExpense({page: page}));
+    },
+    [dispatch],
+  );
+
+  const numberToValidate = useMemo(() => {
+    if (expenseList == null || expenseList === []) {
+      return 0;
+    }
+    if (user?.employee?.hrManager) {
+      const filterList = expenseList?.filter(
+        item => item.statusSelect === Expense.statusSelect.WaitingValidation,
+      );
+      return filterList.length;
+    } else {
+      const filterList = expenseList?.filter(item => {
+        return (
+          user.employee?.managerUser?.id === user.id &&
+          item.statusSelect === Expense.statusSelect.WaitingValidation
+        );
+      });
+      return filterList.length;
+    }
+  }, [
+    expenseList,
+    user.employee?.hrManager,
+    user.employee?.managerUser?.id,
+    user.id,
+  ]);
+
+  const filterOnMode = useCallback(
+    list => {
+      if (list == null || list === []) {
+        return list;
+      } else {
+        if (mode === 'myExpense') {
+          return list?.filter(item => {
+            return item['employee.user.id'] === user.id;
+          });
+        }
+        if (mode === 'toValidate') {
+          if (user?.employee?.hrManager) {
+            return list?.filter(
+              item =>
+                item.statusSelect === Expense.statusSelect.WaitingValidation,
+            );
+          } else {
+            return list?.filter(item => {
+              return (
+                user.employee?.managerUser?.id === user.id &&
+                item.statusSelect === Expense.statusSelect.WaitingValidation
+              );
+            });
+          }
+        }
+      }
+    },
+    [mode, user.employee?.hrManager, user.employee?.managerUser?.id, user.id],
+  );
+
+  const filterOnStatus = useCallback(
+    list => {
+      if (list == null || list === []) {
+        return list;
+      } else {
+        if (selectedStatus.length > 0) {
+          return list?.filter(item =>
+            selectedStatus.find(status => item?.statusSelect === status.key),
+          );
+        } else {
+          return list;
+        }
+      }
+    },
+    [selectedStatus],
+  );
+
+  const filteredList = useMemo(
+    () => filterOnMode(filterOnStatus(expenseList)),
+    [filterOnMode, filterOnStatus, expenseList],
+  );
+
+  return (
+    <Screen removeSpaceOnTop={true}>
+      <HeaderContainer
+        expandableFilter={false}
+        fixedItems={
+          <View style={styles.headerContainer}>
+            <ToggleSwitch
+              styleContainer={[commonStyles.filter, commonStyles.filterSize]}
+              styleToogle={styles.toggle}
+              leftTitle={I18n.t('Hr_MyExpenses')}
+              rightTitle={I18n.t('Hr_ToValidate')}
+              rigthElement={
+                <View style={styles.bubble}>
+                  <Text>{numberToValidate}</Text>
+                </View>
```

Comment:
Create new component in UI package (one commit for that please and don't forget the story)